entc/integration/compose/gremlin-server/Dockerfile

```diff
@@ -1,9 +0,0 @@
-# Copyright 2019-present Facebook Inc. All rights reserved.
-# This source code is licensed under the Apache 2.0 license found
-# in the LICENSE file in the root directory of this source tree.
-
-# Fetch base gremlin server image
-FROM tinkerpop/gremlin-server
-
-# Copy overriden server configuration.
-COPY gremlin-server.yaml tinkergraph-empty.properties /opt/gremlin-server/conf/
```

Comment:
I updated image file thanks for noticing.

I think that docker-compose behaves differently when mounting folder/files into volumes.
When mounting the folder it happens before the "entrypoint" therefore it works.
if I mount it file by file it won't work since the entrypoint script loads the server and then the file is "busy".

Reply:
Well seems like GitHub's actions have their ways.
This will be good enough for now but at some stage, I would like to move the integration tests to docker-compose (meaning the tests will run in a container) this way docker-compose will make sure all services are "up" before starting the tests.